src/content/docs/alerts/create-alert/create-alert-condition/create-nrql-alert-conditions.mdx

```diff
@@ -26,96 +26,20 @@ import queriesnrqlSignalLossUi from 'images/queries-nrql_screenshot-full_signal-
-
-        Click <DNT>**Next**</DNT> when you finish your configuration.
-
-    </Collapser>
-    <Collapser
-        id="step3"
-        title="Step 3. Define your thresholds"
-        >
-    
-        You can define [thresholds](/docs/alerts-applied-intelligence/new-relic-alerts/advanced-alerts/advanced-techniques/set-thresholds-alert-condition/) to determine what will open an incident. Once defined, click <DNT>**Next**</DNT>.
-
-    </Collapser>
-    <Collapser
-        id="step4"
-        title="Step 4. Add details"
-        >
-    
-        Add the details of the alert condition. Add a name to the alert condition to find and use it easily. Then, connect it to a policy. You can select an existing one or create a new policy. 
-
-        If you select <DNT>**New policy**</DNT>:
-        
-          * Add a [meaningful name](/docs/alerts-applied-intelligence/new-relic-alerts/alert-policies/create-edit-or-find-alert-policy/#best-practices-policies) for the policy with a maximum of 64 characters.
-
-          * Select the way you want to [group your incidents](/docs/alerts-applied-intelligence/new-relic-alerts/alert-policies/specify-when-alerts-create-incidents/).
-
-          * Configure the [additional settings](/docs/alerts-applied-intelligence/new-relic-alerts/get-started/your-first-nrql-condition/#additional-settings) if you need to.
-        
-          * Click <DNT>**Save & set up notifications**</DNT>.
-
-
-        If you've selected <DNT>**Existing policy**</DNT>, click <DNT>**Save condition**</DNT> once you've chosen a policy.
-
-    </Collapser>
-    </CollapserGroup>
-
-
-## Create a condition from a chart [#create-chart]
-
-Most of our charts, with the exception of a few older ones, allow you to create a condition from them.
-
-To create a NRQL alert condition from a chart:
-
-1. Go to <DNT>**[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > APM & Services**</DNT> and select an app.
-2. In the left pane, select <DNT>**Metrics explorer**</DNT> and add your configuration.
-3. In the chart's right corner, click the <Icon name="fe-more-horizontal"/> icon and select <DNT>**Create alert condition**</DNT>.
-
-  <img
-    title="Create alert condition from the metrics explorer"
-    alt="Create alert condition from the metrics explorer"
-    src={queriesMetricExplorerAnimation}
-  />
+You can also use one of our alert builders:
 
-4. Follow the steps explained in the [Create a NRQL alert condition from a policy](#alert-condition-from-policy) but be aware that step 1 is a bit different because, now, the query is built with the data you've previously configured.
+* Use <DNT>**Write your own query**</DNT> to build alerts from scratch
+* <DNT>**Use guided mode**</DNT>
```

Comment:
Consider changing to `Use guided mode to choose from options and have your NRQL query built for you`

Reply:
Good call! I fixed this.